src/Event/PrepareUserGroups.php

```diff
@@ -12,6 +12,7 @@
 use Flarum\User\User;
 
 /**
+ * @deprecated beta 13, remove in beta 14. Use the User extender instead.
```

Comment:
These numbers are not correct (anymore :see_no_evil: ). Will fix this in master.